models/commons/src/address/generator.ts

```diff
@@ -5,7 +5,7 @@ const generator = Generator<TAddress>({
   fields: {
     id: fake((f) => f.random.alphaNumeric(8)),
     key: fake((f) => f.lorem.slug(2)),
-    title: fake((f) => f.name.title()),
+    title: fake((f) => f.name.jobTitle()),
```

Comment:
```
Error: [@faker-js/faker]: faker.name.title() is deprecated since v6.1.2 and will be removed in v7.0.0. Please use faker.name.jobTitle() instead.
```